### PR TITLE
Feature/class to module methods

### PR DESCRIPTION
**Pull request by GerryLarios**
Now the methods aren't on a class, they are on the module itself.

### CODE REVIEW

**Review by dgarciaap — APPROVED** [review]

**Review by dsalazar357 — APPROVED** [review]

File: lib/corr.rb [hunk 1]
```diff
@@ -1,44 +1,42 @@
 require "corr/version"
 
 module Corr
-  class Formulas
```
Comment (marioa98) [inline lib/corr.rb:4]: Why do you delete the class inside the module??

**Review by marioa98 — APPROVED** [review]

File: lib/corr.rb (hunk 1 above)
Comment (GerryLarios) [reply lib/corr.rb:4]: The gem doesn't need it. Works fine without the class. Plus short invokes.

File: lib/corr.rb [hunk 2]
```diff
@@ -1,44 +1,42 @@
...
-      for i in 0...n do
-        result += (vector_x[i] - mean_x) * (vector_y[i] - mean_y)
-      end
-      result
+  def self.sum_cov(vector_x, vector_y)
+    n = vector_x.length
+    mean_x = mean(vector_x)
+    mean_y = mean(vector_y)
+    result = 0.0
+    for i in 0...n do
```
Comment (cmagallon1) [inline lib/corr.rb]: Maybe you could've use a reduce with a range to do it in one line

**Review by cmagallon1 — APPROVED** [review]

File: lib/corr.rb (hunk 2 above)
Comment (GerryLarios) [reply lib/corr.rb]: Done, thanks.

**Pull request merged**